patches/chrome-android-BUILD.gn.patch

```diff
@@ -42,31 +42,41 @@ index 3e0a1fe57e55843795740a337a12ea7e8ef0ba3f..221ad122b206479f9ce8b1b115331df2
    }
  
    action_with_pydeps("chrome_android_java_google_api_keys_srcjar") {
-@@ -2311,6 +2316,7 @@ if (current_toolchain == default_toolchain) {
+@@ -1538,6 +1543,9 @@ if (current_toolchain == default_toolchain) {
+       "javatests/src/org/chromium/chrome/browser/toolbar/ToolbarSecurityIconTest.java",
+       "javatests/src/org/chromium/chrome/browser/webapps/WebappLaunchCauseMetricsTest.java",
+     ]
++    sources -= [
++      "javatests/src/org/chromium/chrome/browser/ntp/IncognitoDescriptionViewRenderTest.java",
++    ]
```

Comment:
We probably want to have a one line here as well where what we remove is defined in sources.gni and here we'd have only `sources -= brave_remove_...`. Not sure we have a naming convention for it. cc: @bridiver

Reply:
Changed into a single line.